source/wifi/wifi_hal.c

```diff
@@ -7200,11 +7199,32 @@ INT wifi_getApAssociatedDeviceTidStatsResult(INT radioIndex,  mac_address_t *cli
 
 INT wifi_startNeighborScan(INT apIndex, wifi_neighborScanMode_t scan_mode, INT dwell_time, UINT chan_num, UINT *chan_list)
 {
-    // TODO Implement me!
+    INT status = RETURN_ERR;
+    UINT index;
+    INT Radio_Index;
+    char cmd[256]={0};
```

Comment:
You can use MAX_CMD_SIZE instead of fixed number 256

Reply:
done